recipe/meta.yaml

```diff
@@ -1,37 +1,40 @@
+    sha256: f9a8511da1ef02ec62a77a84fc81f64fa4e80611bf837ed74f0e67e0256f0b45
     patches:
     - patches/003-lin-arm64-compute-lib.patch  # [aarch64]
   - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v24.04.tar.gz  # [aarch64 or arm64]
     sha256: 6d7aebfa9be74d29ecd2dbeb17f69e00c667c36292401f210121bf26a30b38a5  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/373e65b660c0ba274631cf30c422f10606de1618.tar.gz
-    sha256: c205b81f9024952c742e765090556a18c9463fff245753a2afa42c344bd6379d
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/f0f8defe2dff5058391f2a66e775e20b5de33b08.tar.gz
+    sha256: 13bee5b8522177f297e095e3eba5948c1a7ee7a816d19d5a59ce0f717f82cedc
     folder: src/plugins/intel_cpu/thirdparty/onednn
   - url: https://github.com/openvinotoolkit/mlas/archive/d1bc25ec4660cddd87804fcf03b2411b5dfb2e94.tar.gz
     sha256: 0a44fbfd4b13e8609d66ddac4b11a27c90c1074cde5244c91ad197901666004c
     folder: src/plugins/intel_cpu/thirdparty/mlas
-  - url: https://github.com/oneapi-src/oneDNN/archive/37f48519b87cf8b5e5ef2209340a1948c3e87d72.tar.gz  # [win64 or linux64]
-    sha256: 58131e094408460f88bf941977b5206232dc2bc8dbf227250d1e2236b43153a5  # [win64 or linux64]
+  - url: https://github.com/oneapi-src/oneDNN/archive/7ab8ee9adda866d675edeee7a3a6a29b2d0a1572.tar.gz  # [win64 or linux64]
+    sha256: 66363988363744e49fff55f4fcdb72318ff3f35fba6da68302c1662c837c22ac  # [win64 or linux64]
     folder: src/plugins/intel_gpu/thirdparty/onednn_gpu  # [win64 or linux64]
-  - url: https://github.com/oneapi-src/level-zero/archive/4ed13f327d3389285592edcf7598ec3cb2bc712e.tar.gz  # [linux64]
-    sha256: e9fe67f792c7d8f0243eef6ad4e3bb0bb484cc5dfa6790c4fe13289d4fc4df2c  # [linux64]
+  - url: https://github.com/oneapi-src/level-zero/archive/fd459f28041c176df23a0b7b791ff20a7689d237.tar.gz  # [linux64]
+    sha256: e078cae698e368c4e8e57d268f4ef0b766ed8f5548ac8e4fc0a07f4c406bb50d  # [linux64]
     folder: src/plugins/intel_npu/thirdparty/level-zero  # [linux64]
-  - url: https://github.com/intel/level-zero-npu-extensions/archive/d490a130fbb80e600b3aed3886c305abcb60d77c.tar.gz  # [linux64]
-    sha256: 05c6927fdb338efa4cd17a2f5fe3fb0785a5f5c2ef3be02c51dc9e1e9163cd12  # [linux64]
+  - url: https://github.com/intel/level-zero-npu-extensions/archive/518d64125521cd0f8c98d65f9a0fb40013e95d15.tar.gz  # [linux64]
+    sha256: b161b365decd210ff75abb983887178f2847c56dd733e2128a9f5d899a05c7ba  # [linux64]
     folder: src/plugins/intel_npu/thirdparty/level-zero-ext  # [linux64]
   - url: https://github.com/onnx/onnx/archive/refs/tags/v1.15.0.tar.gz
     sha256: c757132e018dd0dd171499ef74fca88b74c5430a20781ec53da19eb7f937ef68
     folder: thirdparty/onnx/onnx
+  - url: https://github.com/gflags/gflags/archive/refs/tags/v2.2.2.tar.gz
```

Comment:
gflags is available in Conda-forge

Reply:
for dev version that is not important, will update the build dependencies later